modules/core/src/main/java/org/apache/ignite/plugin/PluginProvider.java

```diff
@@ -28,11 +28,13 @@
 /**
  * Pluggable Ignite component.
  * <p>
- * Ignite plugins are loaded using JDK {@link ServiceLoader}.
+ * Ignite plugins can be loaded using JDK {@link ServiceLoader} or set up explicitly via
```

Comment:
Please mention the class that should be exposed as a service (i.e. PluginProvider.class itself).